src/Aspire.Dashboard/Model/ValidateTokenMiddleware.cs

```diff
@@ -52,7 +52,11 @@ public async Task InvokeAsync(HttpContext context)
                     qs.Remove("t");
 
                     var newQuerystring = qs.ToString();
-                    context.Response.Redirect($"{context.Request.Path}?{newQuerystring}");
+                    if (!string.IsNullOrEmpty(newQuerystring))
+                    {
+                        newQuerystring = "?" + newQuerystring;
```

Comment:
This isn't new code but we escape this property right?

Reply:
Collection returned from HttpUtility.ParseQueryString encodes correctly in ToString